libraries/liveIntentIdSystem/liveIntentId3System.js

```diff
@@ -0,0 +1,336 @@
+
+// reference to the client for the liQHub
+let cachedClientRef
+
+/**
+ * This function is also used in tests
+ */
+export function resetSubmodule() {
+  cachedClientRef = undefined
+}
+
+resetSubmodule()
+
+window.liQHub = window.liQHub ?? []
+
+function loadConsent() {
+  const consent = {}
+  const usPrivacyString = uspDataHandler.getConsentData();
+  if (usPrivacyString != null) {
+    consent.usPrivacy = { consentString: usPrivacyString }
+  }
+  const gdprConsent = gdprDataHandler.getConsentData()
+  if (gdprConsent != null) {
+    consent.gdpr = gdprConsent
+  }
+  const gppConsent = gppDataHandler.getConsentData();
+  if (gppConsent != null) {
+    consent.gpp = { consentString: gppConsent.gppString, applicableSections: gppConsent.applicableSections }
+  }
+
+  return consent
+}
+
+function initializeClient(configParams) {
+  // only initialize once
+  if (cachedClientRef != null) return cachedClientRef
+
+  const clientRef = {}
+
+  const clientDetails = { name: 'prebid', version: '$prebid.version$' }
+
+  const collectConfig = configParams.liCollectConfig ?? {};
+
+  let integration
+  if (collectConfig.appId != null) {
+    integration = { type: 'application', appId: collectConfig.appId, publisherId: configParams.publisherId }
+  } else if (configParams.distributorId != null) {
+    integration = { type: 'distributor', distributorId: configParams.distributorId }
+  } else {
+    integration = { type: 'custom', publisherId: configParams.publisherId, distributorId: configParams.distributorId }
+  }
+
+  const consent = loadConsent()
+
+  const partnerCookies = new Set(configParams.identifiersToResolve ?? []);
+
+  const collectSettings = { timeout: collectConfig.ajaxTimeout ?? DEFAULT_AJAX_TIMEOUT }
+
+  let identityPartner
+  if (configParams.appId == null && configParams.distributorId != null) {
+    identityPartner = configParams.distributorId
+  } else if (configParams.partner != null) {
+    identityPartner = configParams.partner
+  } else {
+    identityPartner = 'prebid'
+  }
+
+  const resolveSettings = {
+    identityPartner,
+    timeout: configParams.ajaxTimeout ?? DEFAULT_AJAX_TIMEOUT
+  }
+
+  window.liQHub.push({
+    type: 'register_client',
+    clientRef,
+    clientDetails,
+    integration,
+    consent,
+    partnerCookies,
+    collectSettings,
+    resolveSettings
+  })
+
+  // fire default collect request
+  if (configParams.emailHash != null) {
+    window.liQHub.push({ type: 'collect', clientRef, sourceEvent: { hash: configParams.emailHash } })
+  } else {
+    window.liQHub.push({ type: 'schedule_default_collect', clientRef, delay: configParams.fireEventDelay })
+  }
+
+  cachedClientRef = clientRef
+  return clientRef
+}
+
+/**
+ * Create requestedAttributes array to pass to liveconnect
+ * @function
+ * @param {Object} overrides - object with boolean values that will override defaults { 'foo': true, 'bar': false }
+ * @returns {Array}
+ */
+function parseRequestedAttributes(overrides) {
```

Comment:
as this is the same code as in the old module, we can move it into a shared file (in libararies)